File: 7-kyu/a-gift-well-spent/python/solution.py
def buy(x, arr):        
    if len(arr) < 2 or sum(arr) < x:
        return None
    for i in range(len(arr)):
        for j in range(len(arr)):
            if i != j and arr[i] + arr[j] == x:
                return [i, j]

File: 7-kyu/a-gift-well-spent/javascript/solution.js
const buy = (x, arr) => {
  if (arr.length < 2 || arr.reduce((acc, nxt) => acc + nxt, 0) < x) {
    return null;
  }
  for (let i = 0; i < arr.length; i++) {
    for (let j = 0; j < arr.length; j++) {
      if (i != j && arr[i] + arr[j] == x) {
        return [i, j];
      }
    }
  }
  return null;
};
